conf.py

```diff
@@ -1163,7 +1163,11 @@
 # Extra things you want in the pages HEAD tag. This will be added right
 # before </head>
 # (translatable)
-# EXTRA_HEAD_DATA = ""
+EXTRA_HEAD_DATA = """
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
```

Comment:
JQuery should already be loaded at some point on the page, likely at the bottom. You might need to include the `copybutton.js` file at the bottom of the page instead of the top to fix any errors that occur.